dts/common/nordic/nrf9280.dtsi

```diff
@@ -0,0 +1,1207 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <nordic/nrf_common.dtsi>
+
+#include <zephyr/dt-bindings/misc/nordic-nrf-ficr-nrf9230-engb.h>
+#include <zephyr/dt-bindings/misc/nordic-domain-id-nrf9230.h>
+#include <zephyr/dt-bindings/misc/nordic-owner-id-nrf9230.h>
+
+/delete-node/ &sw_pwm;
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpuapp: cpu@2 {
+			compatible = "arm,cortex-m33";
+			reg = <2>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(320)>;
+		};
+
+		cpurad: cpu@3 {
+			compatible = "arm,cortex-m33";
+			reg = <3>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(256)>;
+		};
+
+		cpuppr: cpu@d {
+			compatible = "nordic,vpr";
+			reg = <13>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(16)>;
+			riscv,isa = "rv32emc";
+			nordic,bus-width = <32>;
+
+			cpuppr_vevif_rx: mailbox {
+				compatible = "nordic,nrf-vevif-task-rx";
+				status = "disabled";
+				interrupt-parent = <&cpuppr_clic>;
+				interrupts = <0 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <1 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <2 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <3 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <4 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <5 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <6 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <7 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <8 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <9 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <10 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <11 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <12 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <13 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <14 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <15 NRF_DEFAULT_IRQ_PRIORITY>;
+				#mbox-cells = <1>;
+				nordic,tasks = <16>;
+				nordic,tasks-mask = <0xfffffff0>;
```

Comment:
```suggestion
				nordic,tasks-mask = <0x0000fff0>;
```

Reply:
🤔 is this intended to be different from the configuration on the nRF54h20? Just double checking.

Reply:
This value comes from IPS, where it's different from nRF54H20. However, since the task count is 16, they seem pretty much equivalent here.